pkg/apps/controller/deployer/deployer_controller.go

```diff
@@ -149,6 +149,10 @@ func (c *DeploymentController) handle(deployment *v1.ReplicationController, will
 				return actionableError(fmt.Sprintf("couldn't create deployer pod for %q: %v", deployutil.LabelForDeploymentV1(deployment), err))
 			}
 			updatedAnnotations[deployapi.DeploymentPodAnnotation] = deploymentPod.Name
+			updatedAnnotations[deployapi.DeployerPodCreatedAtAnnotation] = deploymentPod.CreationTimestamp.String()
+			if deploymentPod.Status.StartTime != nil {
+				updatedAnnotations[deployapi.DeployerPodCreatedAtAnnotation] = deploymentPod.Status.StartTime.String()
```

Comment:
the annotation should be `DeployerPodStartedAtAnnotation` not `DeployerPodCreatedAtAnnotation`